packages/store-sync/src/createStoreSync.ts

```diff
@@ -199,14 +201,25 @@ export async function createStoreSync({
   let endBlock: bigint | null = null;
   let lastBlockNumberProcessed: bigint | null = null;
 
-  const storedBlock$ = combineLatest([startBlock$, latestBlockNumber$]).pipe(
+  const indexerLogs$ = indexerUrl
+    ? startBlock$.pipe(
+        mergeMap((startBlock) => {
+          const input = encodeURIComponent(JSON.stringify({ chainId, address, filters }));
+          return fromEventSource<string>(
+            new URL(`/api/logs-live?input=${input}&block_num=${startBlock}&include_tx_hash=true`, indexerUrl),
+          ).pipe(map((messageEvent) => JSON.parse(messageEvent.data) as StorageAdapterBlock));
```

Comment:
I find it more robust+readable to build up URL params like
```suggestion
          return fromEventSource<string>(
            new URL(`/api/logs-live?${new URLSearchParams({
              input: JSON.stringify({ chainId, address, filters }),
              block_num: startBlock.toString(),
              include_tx_hash: "true",
            })}`, indexerUrl),
          ).pipe(map((messageEvent) => JSON.parse(messageEvent.data) as StorageAdapterBlock));
```